stdlib/sysconfig.pyi

```diff
@@ -15,11 +16,15 @@ __all__ = [
     "parse_config_h",
 ]
 
+@overload
+@deprecated("SO is deprecated, use EXT_SUFFIX")
```

Comment:
sysconfig.py doesn't mention this, can you link to where it's indicated that "SO" is deprecated?

Reply:
Looks like support has been removed in Python 3.11
https://github.com/python/cpython/pull/91671/files#diff-d593bd299ba58e440ba411ffa0640ccd9d20d518b0cf2644ed4bdb75a82a3e70L670-L673

I'll add this to the deprecation comment now that I found this.

Reply:
Thanks, I only looked at the main branch.